apps/chat/src/chat/chat.controller.ts

```diff
@@ -1,33 +1,36 @@
-import {
-  GetChatListDto,
-  GetChatRoomDto,
-  GetChatRoomsDto,
-} from '@apps/api-gateway/src/chat/dto/req.dto';
-import { Controller } from '@nestjs/common';
+import { Controller, HttpStatus, UseInterceptors } from '@nestjs/common';
 import { MessagePattern, Payload } from '@nestjs/microservices';
 import { MESSAGE } from '@shared/constants/message-pattern';
+import { SetResponse } from '@shared/decorator/set-response.decorator';
+import { LoggingInterceptor } from '@shared/interceptor/message-logging.interceptor';
+import { TransformInterceptor } from '@shared/interceptor/transform.interceptor';
 import { ChatService } from './chat.service';
-import { CreateChatRoomDto } from './dto/req.dto';
+import { CreateChatRoomDto, GetChatListDto, GetChatRoomDto, GetChatRoomsDto } from './dto/req.dto';
 
+@UseInterceptors(LoggingInterceptor, TransformInterceptor)
 @Controller()
 export class ChatController {
   public constructor(private readonly chatService: ChatService) {}
 
+  @SetResponse(MESSAGE.CHAT.CREATE_CHAT_ROOM.cmd, HttpStatus.OK)
```

Comment:
```suggestion
  @SetResponse(MESSAGE.CHAT.CREATE_CHAT_ROOM.cmd, HttpStatus.CREATED)
```

명확하게 CREATED (201) 로 명시되는게 좋을 것 같습니다~!